purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -693,21 +695,29 @@ internal class BillingWrapper(
                 BillingClient.BillingResponseCode.FEATURE_NOT_SUPPORTED,
                 BillingClient.BillingResponseCode.BILLING_UNAVAILABLE,
                 -> {
-                    val message =
-                        BillingStrings.BILLING_UNAVAILABLE.format(billingResult.toHumanReadableDescription())
-                    log(LogIntent.GOOGLE_WARNING, message)
+                    val originalErrorMessage = billingResult.toHumanReadableDescription()
+
+                    val error = if (billingResult.debugMessage == IN_APP_BILLING_LESS_THAN_3_ERROR_MESSAGE) {
```

Comment:
how can this even happen?

Reply:
oh, wait, do you get this with something completely unrelated? If so, we should leave a comment, that's not something a maintainer would expect

Reply:
yeah, the debug message is useless, API version 3 is actually a bit old (2012) lol

I will add a comment to make sure the next person reading this understands why